Kernel/Heap/kmalloc.cpp

```diff
@@ -30,7 +30,11 @@ static constexpr size_t CHUNK_SIZE = 64;
 #endif
 
 #define POOL_SIZE (2 * MiB)
-#define ETERNAL_RANGE_SIZE (4 * MiB)
+#define ETERNAL_HEAPS_PREALLOCATED_MEMORY_SIZE (100 * KiB)
+// FIXME: Clang doesn't build if we set this value to be 3 MiB. We should probably
+// find a way to add more eternal heaps before initializing the MemoryManager,
+// so we don't need to expand it further.
+#define FIRST_ETERNAL_RANGE_SIZE (4 * MiB)
```

Comment:
How do you mean "not build"? How did you test this code, because 4 MiB is quite huge already?